central/sensor/service/connection/connection_impl.go

```diff
@@ -578,6 +581,24 @@ func (c *sensorConnection) Run(ctx context.Context, server central.SensorService
 		}
 	}
 
+	// TODO: Temporary block from https://github.com/stackrox/stackrox/pull/7755
```

Comment:
This makes me thing that maybe the best way to start the stream of changes in Sensor is to have an integration test where fake central sends the deduper state. Then we won't need to copy snippets around.